src/args.rs

```diff
@@ -61,11 +61,25 @@ pub struct Args {
 #[derive(Parser, Debug)]
 pub enum SubCommands {
     /// Render the context without running a query against the model.
-    View(ViewSubCommand)
+    View(ViewSubCommand),
+    /// Set or get configuration values.
+    Config(ConfigSubCommand),
 }
 
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 pub struct ViewSubCommand {
     // You can add options and arguments specific to the `view` subcommand here.
 }
+
+#[derive(Parser, Debug)]
+#[command(author, version, about = "Set or get configuration values", long_about = None)]
+pub struct ConfigSubCommand {
+    /// Set a configuration value in the application.
+    #[arg(short, long)]
+    pub set: Option<String>,
```

Comment:
♻️ It might be good to expand this docs comment to include an example of what the command looks like when being used, I personally didn't get it right first try. 

![image](https://github.com/divanvisagie/chat-gipity/assets/2023086/91f922dd-f2b4-4bc4-b9d8-8eddd95b514e)